Player/Console/PlayerCommand.php

```diff
@@ -113,10 +114,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
     }
 
-    private function createClient()
+    private function createClient(OutputInterface $output)
     {
         return new GuzzleClient([
             'cookies' => true,
+            'debug' => $output->isDebug() && $output instanceof ConsoleOutput
```

Comment:
nice!
what about moving this 3-liner in a variable so it would be easier to read?